<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupDialog</class>
 <widget class="QDialog" name="PopupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurar Iteraciones</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QLabel {
        color: #ffffff;
        font-size: 18px;
        font-weight: bold;
    }
    QLineEdit {
        padding: 8px;
        border: 2px solid #bdc3c7;
        border-radius: 6px;
        background-color: white;
        font-size: 14px;
        color: black;
    }
    QLineEdit:focus {
        border: 2px solid #3498db;
    }
    QPushButton {
        padding: 8px 20px;
        border-radius: 6px;
        font-size: 14px;
        font-weight: bold;
        color: white;
        min-width: 100px;
    }
    QPushButton#pushButton {
        background-color: #3498db;
        border: none;
    }
    QPushButton#pushButton:hover {
        background-color: #2980b9;
    }
    QPushButton#pushButton_2 {
        background-color: #e74c3c;
        border: none;
    }
    QPushButton#pushButton_2:hover {
        background-color: #c0392b;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>¿Cuántas iteraciones desea realizar?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">
       font-weight: normal;
       color: #7f8c8d;
      </string>
     </property>
     <property name="text">
      <string>Ingrese un número entre 1 y 1000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Número de iteraciones</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
